hrm-service/src/contact/contact.ts

```diff
@@ -45,16 +45,17 @@ export type SearchContact = {
   csamReports: CSAMReportEntry[];
 };
 
-type CreateContactPayloadWithFormProperty = Omit<NewContactRecord, 'rawJson'> & {
+export type CreateContactPayloadWithFormProperty = Omit<NewContactRecord, 'rawJson'> & {
```

Comment:
From https://github.com/techmatters/flex-plugins/blob/04b33229f9279f652116a310e4c551918eefc145/plugin-hrm-form/src/services/ContactService.ts#L222, `(property) form: ContactRawJson`.

Maybe we could refactor and change this name eventually? Or do you think is best to keep it around?

Reply:
I agree with doing this. Should we have a ticket so we don't forget?